nodejs/week1/avg.js

```diff
@@ -0,0 +1,15 @@
+const inputValues = process.argv.slice(2);
+const numbersFromValues = inputValues.map((item) => Number(item));
+
+if (numbersFromValues.some((num) => isNaN(num))) {
+	console.error("Please, enter valid numbers");
+	process.exit(1);
```

Comment:
Very nice to use this function! This is exactly how we should let the user know that the code did not succeed.